### PR TITLE
Prevent the OpenDocument preview generator from trying to open empty files.

### PR DESCRIPTION
**Pull request by rotdrop**
Rationale: does not make sense, and triggers a deprecation error in
\ZipArchive.

Signed-off-by: Claus-Justus Heine <himself@claus-justus-heine.de>

### CODE REVIEW

File: lib/private/Preview/Bundled.php [hunk 1]
```diff
@@ -32,6 +32,11 @@
  */
 abstract class Bundled extends ProviderV2 {
 	protected function extractThumbnail(File $file, $path): ?IImage {
+
+		if ($file->getSize() == 0) {
```
Comment (CarlSchwan) [inline lib/private/Preview/Bundled.php:36]:
```suggestion
		if ($file->getSize() === 0) {
```

**Review by CarlSchwan — APPROVED** [review]
Make sense to me

**Review by juliusknorr — COMMENTED** [review]
Looks good, mind to address the small suggestion to do a strict comparison?

**Review by juliusknorr — APPROVED** [review]

**Comment by szaimen** [comment]
rebase: https://github.com/nextcloud/server/pull/37772

**Pull request closed without merging**